Scripts/AnalyzeDatabank/calc_FormFactors.py

```diff
@@ -206,8 +206,8 @@
             system['WARNINGS'] is not None and
             'GROMACS_VERSION' in system['WARNINGS'] and
             system['WARNINGS']['GROMACS_VERSION'] == 'gromacs3'):
-            trjconvCOMMAND = '/home/osollila/Programs/gromacs/gromacs402/bin/trjconv'
-            makendxCOMMAND = '/home/osollila/Programs/gromacs/gromacs402/bin/make_ndx'
+            trjconvCOMMAND = 'trjconv'
+            makendxCOMMAND = 'make_ndx'
         else:
```

Comment:
similar to me previous comment (https://github.com/NMRLipids/Databank/pull/201/files/80a00491045c906ce73eaa232173d50003047676..9ce6527a91efe20944b95395bb3244fbcdde0c4b#r1831033418) we should move these various commands existing in different scripts into a single file

Reply:
Don't understand what I should do here.

Reply:
Both `calcOrderParameters` and `calc_FormFactors` contain the same commands like `trjconvCOMMAND` . These commands are individually defined in each of these scripts.

I personally would prefer if we had a `config` file in the databank (could be a python file, json file, etc) that would unify these definitions. Then all we will need to do is to read the `trjconvCOMMAND` (and other similar definitions) from this single config file.

Reply:
We can handle this in a next iteration.